Algorithm.FSharp/QuantConnect.Algorithm.FSharp.fsproj

```diff
@@ -10,7 +10,7 @@
     <RootNamespace>QuantConnect.Algorithm.FSharp</RootNamespace>
     <AssemblyName>QuantConnect.Algorithm.FSharp</AssemblyName>
     <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
-    <TargetFSharpCoreVersion>4.3.1.0</TargetFSharpCoreVersion>
+    <!--<TargetFSharpCoreVersion>4.3.1.0</TargetFSharpCoreVersion>-->
```

Comment:
Does removing this still work in a fresh install? How does Nuget know what to restore?

Reply:
I have added the F# runtime as a nuget package in packages.config - I believe that is the correct way of referencing F# runtime going forward